jekyll/__glossary/Pipelines.md

```diff
@@ -2,6 +2,6 @@
 term: Pipeline
 --- 
 
-A pipeline is an object that you may configure and run on the CircleCI platform. A pipeline can be triggered through webhooks when code is pushed to your git repository, or when you use the CircleCI API. The pipeline's configuration includes workflows and jobs that may be run with other elements like parameters and orbs. 
+CircleCI pipelines are the most high-level unit of work, encompassing a project's full `.circleci/config.yml` file, including workflows, which coordinate jobs. Pipelines have a fixed, linear lifecycle, and are associated with a specific actor. Pipelines run when a change is pushed to a project that has a CircleCI configuration file included, and can also be scheduled, triggered manually through the CircleCI app, or using the API.
```

Comment:
@sarahtejas I've mashed together all the pipelines definitions you pointed me to and have come up with this, which is certainly not perfect! Any help in making this flow a bit better would be much appreciated!